docker/Dockerfile.dmqnode

```diff
@@ -14,13 +14,17 @@ RUN docker/build.sh
 ARG DIST=xenial
 # For now plain ubuntu image is used as base for simplicity. It certainly can be
 # optimized to bare minimum but right now it is not important:
-FROM sociomantictsunami/runtimebase:$DIST-v6
+FROM sociomantictsunami/runtimebase:$DIST-v7
+
+# Set up directories and install the node
 COPY --from=builder /project/build/production/pkg/ /packages/
-COPY docker/install.sh /tmp/
-RUN /tmp/install.sh && rm /tmp/install.sh
-# Update node folder with relevant config to finalize installation
-COPY ./doc/etc/config.docker.ini /srv/dmqnode/dmqnode-0/etc/config.ini
+COPY docker/install.sh /
+RUN /install.sh && rm /install.sh
```

Comment:
Here is where I think it might be nice to have the important info easy to see, you could do something like:
```dockerfile
RUN /install.sh pkg1 pkg2 pk3 && rm /install.sh
```
But is just a suggestion in case you wanted to gather all the important info in the dockerfile and leave `install.sh` only with the very uninteresting stuff.

Reply:
I think it's fine as it is. One has to look into the file in any case to understand what's going on. And `install.sh` really only installs the node and sets up some dirs.